app/controllers/api/v1/api_keys_controller.rb

```diff
@@ -5,12 +5,16 @@ class Api::V1::ApiKeysController < Api::BaseController
   def show
     authenticate_or_request_with_http_basic do |username, password|
       user = User.authenticate(username, password)
-      if user
+      otp = request.headers['HTTP_OTP']
+      if user&.mfa_api_authorized?(otp)
```

Comment:
why aren't we using `before_action :verify_with_otp` here?

Reply:
`verify_with_otp` requires `@api_user` set. We only require username & password in `ApiKeys#show` and `Sessions#create`.